dashboard/src/actions/charts.ts

```diff
@@ -2,6 +2,7 @@ import { Dispatch } from "redux";
 import { ThunkAction } from "redux-thunk";
 import { ActionType, createAction } from "typesafe-actions";
 
+import { axios } from "../shared/Auth";
```

Comment:
One concern about this is that we'll be sending the Kubernetes API token to services that don't use it. We should try to avoid that if possible, to minimise the exposure of those sensitive tokens. It will also be an issue if we need a separate header for these APIs in the future.

How about we setup a different axios instance that shares the interceptor for the XHR error, but doesn't send the token?